runtime/lua/lsp/client.lua

```diff
@@ -0,0 +1,304 @@
+local uv = require('luv')
```

Comment:
I'm hoping to not have to use `luv` soon. Once we get the jobs update (which I assume will happen before I finish this :) ) then I will use nvim's included job functionality, rather than a hacked together interpretation.